.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,3 +14,4 @@ If tests do fail, click on the red `X` to learn why by reading the logs.
 Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
 
 Thanks!
+@publiclab/is-reviewers please review my PR :smile:.
```

Comment:
I think this probably won't work because the new contributor won't be a team member, so Github doesn't let them mention groups...

Reply:
I had edited the pr body of a newcomer and i could mention is-reviewers. Now we don't know if it worked because I edited or otherwise. So let's experiment. We can change things later.